pipeline/test/services/service-cluster/testEndpoints.sh

```diff
@@ -64,6 +57,6 @@ testEndpointProtected OpenSearch "https://${opensearch_subdomain}.${ops_domain}/
 
 testEndpointProtected OpenSearchDashboards "https://${opensearch_dashboards_subdomain}.${base_domain}/" 302
 
-if [[ "${enable_thanos}" == "true" ]] && [[ "${enable_thanos_receiver}" == "true" ]]; then
+if [[ "${enable_thanos_receiver}" == "true" ]]; then
     testEndpointProtected OpenSearchDashboards "https://${thanos_subdomain}.${ops_domain}/" 401
```

Comment:
Please rename it to ThanosReceiver instead of OpenSearchDashboards.

Reply:
Done